host_core/lib/host_core/actors/actor_supervisor.ex

```diff
@@ -41,35 +43,91 @@ defmodule HostCore.Actors.ActorSupervisor do
       Logger.debug("Start actor request received for #{oci}", oci_ref: oci)
 
       source = HostCore.Policy.Manager.default_source()
+      claims_result = get_claims(bytes, oci)
 
       with {:ok, {pid, _}} <- VirtualHost.lookup(host_id),
            config <- VirtualHost.config(pid),
            labels <- VirtualHost.labels(pid),
-           {:ok, claims} <- get_claims(bytes, oci),
+           {:ok, claims} <- claims_result,
```

Comment:
Did dialyzer recommend this change? By putting the call up on line 46, it gets invoked every single time, regardless of whether  `config` or `labels` return null

Reply:
I actually did this change so that we could publish the claims in more scenarios when the actor start fails, if I remember correctly